Added `rubocop-changed` extension to documentation

dukaev opened the pull request:
**Replace this text with a summary of the changes in your PR.
The more detailed you are, the better.**

-----------------

Before submitting the PR make sure the following are checked:

* [ ] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [ ] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [ ] Feature branch is up-to-date with `master` (if not - rebase it).
* [ ] Squashed related commits together.
* [ ] Added tests.
* [ ] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [ ] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

bbatsov commented on docs/modules/ROOT/pages/extensions.adoc:
```diff
@@ -99,6 +99,8 @@ Upstream best practices and coding conventions for downstream (e.g. Debian packa
 * https://github.com/DmitryTsepelev/rubocop-graphql[rubocop-graphql] -
 GraphQL-specific analysis
+* https://github.com/dukaev/rubocop-changed[rubocop-changed] -
+Reduced CI time by analysis only changed files
```
analysis -> analyzing

dukaev commented:
@bbatsov thank you for comment.
I updated [readme](https://github.com/dukaev/rubocop-changed): added that is required and [similar gems](https://github.com/dukaev/rubocop-changed/#similar-gems).
